### ISSUE TITLE
Compatibility with PHP 7.4

### ISSUE THREAD

**Issue by esurov** [saintsystems/odata-client-php#134]
UPDATE: Please check the PR that fixes this issue: https://github.com/saintsystems/odata-client-php/pull/135

The package states to support `PHP 7.4` in the `composer.json` and also in the `.travis.ci` (the builds don't run though)

```
vendor/bin/phpunit -c phpunit.xml
PHPUnit 9.6.9 by Sebastian Bergmann and contributors.

..PHP Fatal error:  Declaration of SaintSystems\OData\Entity::offsetExists(SaintSystems\OData\mixed $offset): bool must be compatible with ArrayAccess::offsetExists($offset) in /var/development/odata-client-php/src/Entity.php on line 820
```

**Comment by n8rowley** [comment]
If we could get this fixed, that would be great. 

Does anyone know the latest version of this package that is still compatible with PHP7.4?

**Comment by gRegorLove** [comment]
@n8rowley Looks like it's the previous version, 0.6.2. That's working for me at first glance under PHP7.4

It looks like the Travis CI builds haven't been running for a while. I'll open a separate issue about that so ideally the build process will catch these types of issues.

**Comment by anderly** [comment]
Should be fixed now. Github Actions build added with matrix of PHP 7.3, 7.4, 8.0, 8.1 and 8.2.

**Comment by anderly** [comment]
This is available in [Release 0.6.4](../releases/tag/0.6.4)

**Comment by n8rowley** [comment]
Looks like we missed one instance. See #139

**Comment by anderly** [comment]
@n8rowley, this is fixed in [Release 0.6.5](../releases/tag/0.6.5)
